File: MATTHEW/leetcode/AIcoding/4.GPT-4o/javascript_solutions/2541-minimum-operations-to-make-array-equal-ii.js
function minOperations(nums) {
    const n = nums.length;
    nums.sort((a, b) => a - b);
    let minOps = Infinity;

    for (let i = 0; i < n; i++) {
        let ops = 0;
        for (let j = 0; j < n; j++) {
            ops += Math.abs(nums[j] - nums[i]);
        }
        minOps = Math.min(minOps, ops);
    }

    return minOps;
}

File: leetcode/AIcoding/4.GPT-4o/c_solutions/2541-minimum-operations-to-make-array-equal-ii.c
int minOperations(int* nums, int numsSize) {
    int min = nums[0], max = nums[0];
    for (int i = 1; i < numsSize; i++) {
        if (nums[i] < min) min = nums[i];
        if (nums[i] > max) max = nums[i];
    }
    return (max - min) / 2 + (max - min) % 2;
}